src/pybind/mgr/dashboard/frontend/src/app/shared/components/notifications-sidebar/notifications-sidebar.component.ts

```diff
@@ -31,6 +30,8 @@ import { PrometheusNotificationService } from '~/app/shared/services/prometheus-
 import { SummaryService } from '~/app/shared/services/summary.service';
 import { TaskMessageService } from '~/app/shared/services/task-message.service';
 
+const POLLING_INTERVAL = 5000; // milliseconds
```

Comment:
The pooling interval of 5 sec is already defaulted in session of browser, do we need to define this?